daprdocs/content/en/developing-applications/building-blocks/conversation/howto-conversation-layer.md

```diff
@@ -0,0 +1,9 @@
+---
+type: docs
+title: "How-To: Migrate using the conversation API"
+linkTitle: "How-To: Migrate"
+weight: 2000
+description: "Learn how to use the conversation API to migrate between large language model servers and toolkits"
```

Comment:
`Learn how to abstract the complexities of interacting with large language models`